crates/wasmtime-provider/Cargo.toml

```diff
@@ -35,7 +35,10 @@ anyhow = "1.0"
 thiserror = "1.0"
 cfg-if = "1.0.0"
 parking_lot = "0.12"
-
+serde = { version = "1.0", features = ["derive"] }
+hex = "0.4.3"
+smallvec = "1.10.0"
+console = "0.15.5"
```

Comment:
These should be developer dependencies, since they are required only by the example